GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -68,14 +69,29 @@ void OnGUI()
         private void OnGuiCreateBundleSelect() 
         {
             EditorGUILayout.LabelField("AssetBundle Creation", EditorStyles.boldLabel);
-            EditorGUILayout.LabelField("Use the Unity Asset Bundle Browser to select your game's main scene " + 
-                "and bundle it (and its dependencies) into an AssetBundle file.", EditorStyles.wordWrappedLabel);
+            EditorGUILayout.LabelField("Use the Unity AssetBundles Browser to select your game's main scene " + 
+                                       "and bundle it (and its dependencies) into an AssetBundle file.", EditorStyles.wordWrappedLabel);
             EditorGUILayout.Space();
-            EditorGUILayout.LabelField(string.Format("AssetBundle Browser version: {0}", "not found"), EditorStyles.wordWrappedLabel);
+            EditorGUILayout.LabelField($"AssetBundle Browser version: { AssetBundleBrowserClient.BundleBrowserVersion }",  EditorStyles.wordWrappedLabel);
             EditorGUILayout.Space();
-            GUILayout.Button ("Download AssetBundle Browser", GUILayout.Width(ButtonWidth));
-            EditorGUILayout.Space();
-            GUILayout.Button ("Open AssetBundle Browser", GUILayout.Width(ButtonWidth));
+            
+            // Allow the developer to open the AssetBundles Browser if it is present, otherwise ask them to download it
+            if (AssetBundleBrowserClient.BundleBrowserIsPresent())
+            {
+                var openAssetBundleBrowser = GUILayout.Button ("Open AssetBundles Browser", GUILayout.Width(ButtonWidth));
+                if (openAssetBundleBrowser)
+                {
+                    AssetBundleBrowserClient.DisplayAssetBundleBrowser();
+                }
+            } else 
+            {
+                var downloadAssetBundleBrowser =  GUILayout.Button ("Download AssetBundles Browser", GUILayout.Width(ButtonWidth));
```

Comment:
Link to the github releases page directly to save some time?

Let's have 2 buttons maybe:
"Open Asset Bundle Browser Documentation" (the link you have now)
and
"Download Asset Bundle Browser from GitHub" (the github releases page)